app/models/loader/handlers/password.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Loader
+  module Handlers
+    # extends the policy loader to store role credentials (passwords). This occurs
+    # after the initial policy load, since variable values are not part of the policy load
+    module Password
+  
+      # When a public key is encountered in a policy, it is saved here. It can't be written directly into
```

Comment:
This comment and the corresponding one in `PublicKey` both seem to have some problems: this comment mentions "public key", but the module is for loading passwords.

I know you just copied-and-pasted them from the original, but can you fix them, please?